sdk/digitaltwins/Azure.ResourceManager.DigitalTwins/src/autorest.md

```diff
@@ -0,0 +1,15 @@
+# Generated code configuration
+
+Run `dotnet msbuild /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+payload-flattening-threshold: 1
```

Comment:
@allenjzhang  If I remove this, I get a lot of changes to the generated code that don't make sense to me. For instance, I'll lose description and get tags. I copied this from the readme.md where the swagger is. What does it do? Where can I read up more on options? What options do you recommend?